### ISSUE TITLE
Crash when i use multiline comment in SQL

### ISSUE THREAD

**Issue by swpigris81** [zendesk/maxwell#1628]
One SQL like below：
```
/*
 Navicat Premium Data Transfer

 Source Server         : MySQL-Test
 Source Server Type    : MySQL
 Source Server Version : 50728
 Source Host           : 192.168.1.2:3306
 Source Schema         : sys

 Target Server Type    : MySQL
 Target Server Version : 50728
 File Encoding         : 65001

 Date: 18/01/2021 15:12:04
*/

-- ----------------------------
-- Procedure structure for create_synonym_db
-- ----------------------------
DROP PROCEDURE IF EXISTS create_synonym_db
```

**Maxwell crashed and throw Exception like this:**

com.zendesk.maxwell.schema.ddl.MaxwellSQLSyntaxError: DROP
	at com.zendesk.maxwell.schema.ddl.MysqlParserListener.visitErrorNode(MysqlParserListener.java:93)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:41)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at org.antlr.v4.runtime.tree.ParseTreeWalker.walk(ParseTreeWalker.java:52)
	at com.zendesk.maxwell.schema.ddl.SchemaChange.parseSQL(SchemaChange.java:95)
	at com.zendesk.maxwell.schema.ddl.SchemaChange.parse(SchemaChange.java:107)
	at com.zendesk.maxwell.schema.ddl.DDLParserTest.parse(DDLParserTest.java:31)
	at com.zendesk.maxwell.schema.ddl.DDLParserTest.testDropProcedure2(DDLParserTest.java:431)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)

**Comment by osheroff** [comment]
Hi, 
I can't seem to replicate your error.  is it possible that you can show me the output of `show binlog events` for the statement that crashed maxwell?  also, what version of maxwell?

**Comment by swpigris81** [comment]
The output of `show binlog events` like below, and my maxwell version is maxwell-1.23.5
```
mysql> show binlog events in 'mysql-bin.000038' from 219293565 limit 20;
+------------------+-----------+----------------+-----------+-------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Log_name         | Pos       | Event_type     | Server_id | End_log_pos | Info                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+------------------+-----------+----------------+-----------+-------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| mysql-bin.000038 | 219293565 | Query          |         1 |   219294136 | use `ds_order`; /*
 Navicat Premium Data Transfer

 Source Server         : MySQL-Test
 Source Server Type    : MySQL
 Source Server Version : 50728
 Source Host           : 192.168.1.2:3306
 Source Schema         : sys

 Target Server Type    : MySQL
 Target Server Version : 50728
 File Encoding         : 65001

 Date: 18/01/2021 15:12:04
*/

-- ----------------------------
-- Procedure structure for create_synonym_db
-- ----------------------------
DROP PROCEDURE IF EXISTS create_synonym_db |
```

**Comment by swpigris81** [comment]
And I tested in maxwell-1.29.2, Maxwell crashed and throw the same Exception like above.

**Comment by osheroff** [comment]
fixed in 1.30.0
